modules/gitops-argo-cd-dynamic-scaling-in-web-console.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// * declarative_clusterconfig/sharding-clusters-across-argo-cd-application-controller-replicas.adoc
+
+:_mod-docs-content-type: PROCEDURE
+
+[id="gitops-argo-cd-dynamic-scaling-in-web-console_{context}"]
+= Enabling dynamic scaling of shards in the web console
+
+You can enable dynamic scaling of shards by using the {OCP} web console.
+
+.Prerequisites
+* You have access to the cluster with `cluster-admin` privileges.
+* You have access to the {OCP} web console.
+* You have installed the {gitops-title} Operator in your cluster.
+
+.Procedure
+
+. In the *Administator* perspective of the {OCP} web console, go to *Operators* -> *Installed Operators*.
+
+. From the the list of *Installed Operators*, select the {gitops-title} Operator, and then click on the *ArgoCD* tab.
```

Comment:
There's a few instances of this in the PR, but IBMSG says to avoid using Click on: https://www.ibm.com/docs/en/ibm-style?topic=grammar-prepositions

Just use Click instead

Reply:
ACk.
addressed